src/test/java/apoc/hashing/FingerprintingTest.java

```diff
@@ -0,0 +1,75 @@
+package apoc.hashing;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.harness.junit.Neo4jRule;
+import org.neo4j.helpers.collection.Iterators;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class FingerprintingTest {
+
+    @Rule
+    public Neo4jRule neo4j = new Neo4jRule()
+            .withFunction(Fingerprinting.class);
+
+    @Test
+    public void fingerprintNodeWithArrayProperty() {
+        GraphDatabaseService db = neo4j.getGraphDatabaseService();
+        db.execute("CREATE (:Person{name:'ABC',emails:['aa@bb.de', 'cc@dd.ee'], integers:[1,2,3], floats:[0.9,1.1]})");
+
+        String value = Iterators.single(db.execute("MATCH (n) RETURN apoc.hashing.fingerprintNodeOrRel(n) AS hash").columnAs("hash"));
```

Comment:
you can just create + fingerprint without doing the extra match, makes it more 1:1